portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/administration/users/gdpr/GDPR.testcase

```diff
@@ -455,13 +455,93 @@ definition {
 		}
 	}
 
-	@description = "This test anonymizes a staged and a live entry from an applciation."
-	@ignore = "Test Stub"
+	@description = "This test anonymizes a staged and a live entry from an applciation, test with WC entries."
 	@priority = "3"
-	test CanAnonymizeASingleStagedAndLiveEntryFromAnApplication {
+	test CanAnonymizeASingleStagedAndLiveEntryFromWebContent {
+		task ("Given log in as a new user, create a site enable local staging") {
+			User.logoutPG();
+
+			User.firstLoginUI(
+				password = "test",
+				userEmailAddress = "userea@liferay.com");
+
+			JSONGroup.addGroup(groupName = "Site Name");
+
+			JSONLayout.addPublicLayout(
+				groupName = "Site Name",
+				layoutName = "Staging Test Page");
+
+			JSONUser.addUserToSite(
+				groupName = "Site Name",
+				userEmailAddress = "userea@liferay.com");
+
+			JSONStaging.enableLocalStaging(groupName = "Site Name");
+		}
+
+		task ("Given add the staged entry with web content and publish to live this entry") {
```

Comment:
We not using multi Given when apply the BDD format, we can either put them in one given block or nested by using And.
In order to put the Given together, we can use a description like this e.g.:Publish the staged web content entries to live as a new admin user.